src/cluster/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
```

Comment:
Does this need to be a separate crate from `clusterd`? I'd probably just move the `clusterd` binary into this crate.

Reply:
@benesch this crate is _above_ the storage and compute crates, so it can't be merged